### DISCUSSION TITLE
I can help establish Stable Business Development team

### DISCUSSION THREAD

**Discussion by AnonymousMaximus2** [WeAreMahsaAmini/Core#115] [Ideas]
Hi,

I can help you in the following matters:

1- Establishing Not for profit organization in USA that
2- Helping on Legal works, including copyrights, agreements, trademark, patent, website ownership, intellectual property, ... 
3- Funding projects (Grant funding from US Government, Bank funding, Financial markets funding, Crowd Funding,...) 
4- establishing broadcasting team for production of media content such as reels, stories, animation, film production, documentary,...
5- Advertising on social media (Instagram, Facebook, google, ...) 
6- I have connection with Instagram account holders in accumulated 10 Million followers.
7- Email marketing, .....
7- Developing funding campaign for not for profit or human rights projects

based on more than 40 years of experience I can suggest you:

**Without money the world can not hear you**

1- Establish your not for profit company in USA

2- Open bank account, and create accounting reporting system, 

3- Get your company IRS code and Section 501(c)(3) status to become exempt from paying tax

4- With Section 501(c)(3) status you can start online funding campaign for expenses of your projects

5- With the money you collected hire some experienced full time employees to produce content and spread the word.

**I am here to help,**

About me:

Please dont ask for my personal life and personal info, 

I have been in most Iranian Large multi billion $USD projects as adviser but because of not engaging in their international money laundering schemes, 

I lost almost everything ...

For some security reasons and problems that I have with Iranian government, I can not disclose,

but here is some useful information that can help you understand how I can help:

I have been in Financial markets since 1995, and my main profession is Investment Banking,

I established my first business development company in Iran in mid of 2000 
Then I established my UK advisory company in 2002 
After few years I established my US based company in 2010

I have been in Business Development for last 20 years mostly in Iran,

**Comment by syavash** [comment]
I know you want to be anonymous and I absolutely respect it. What's the best way to move this forward?

Could you please contact me on Telegram if possible? https://t.me/siavash

I'm open to any other form of communication.

**Comment by AnonymousMaximus2** [comment]
I prefer online chat platforms

I created Zulipchat organization here

https://wearemahsaamini.zulipchat.com

If you send me your Email, I can invite you there for better organized chat , it is something like slack.

**Reply by HeyItsMehMeh** [reply]
Hey , can you invite me please? heyitsmeh@writeme.com

**Comment by AnonymousMaximus2** [comment]
There are many ideas in this group about NFT, Metaverse, Crypto currency, money donation, funding iranian people, .....

Who will be responsible for Anti Money laundering and legal aspects of this idea ?

You never and can not do successful job until you take serious my idea to establish your not for profit organization:

you need at least 1 person can tell you about difference between legal acts and illegal acts, copyrights, Anti money laundering , tax paying, having separate bank account, ..... responsible persons for collecting money from people and spending money for elligible people.

**Reply by armantorkzaban** [reply]
@AnonymousMaximus2 we are already a NG-NPO and we know a bit about the field. But we can still benefit from your expertise. Look at our work and check if you would like to work with us: https://github.com/tcfev

**Comment by Pink3lephant** [comment]
Hi! @AnonymousMaximus2 I'm curious about the outcome of this discussion, I also do think it's very important and we need to start the process sooner than later 🙌  

Can you please reach out to me at freeinternettech@gmail.com to discuss it further?

**Comment by JohnnyCheungBAK** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
